client/driver/raw_exec.go

```diff
@@ -247,7 +255,7 @@ func (h *rawExecHandle) Update(task *structs.Task) error {
 }
 
 func (h *rawExecHandle) Exec(ctx context.Context, cmd string, args []string) ([]byte, int, error) {
-	return execChroot(ctx, "", cmd, args)
+	return execScript(ctx, h.taskDir.Dir, h.taskEnv, cmd, args)
```

Comment:
Well there was no good reason to have the exact same code here and in UniversalExecutor.Exec, so I at least combined them.

I could have raw_exec (and rkt, the other driver that behaves this way) call the executor, but honestly it's just more runtime overhead to run the exact same code. If it actually has a functional difference in the future it will be easy to change then.